ob_intern/Extension/Gradient.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Gradient.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 15/6/2566 BE.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+
+    public func setGradient(colorOne: UIColor, colorTwo: UIColor) {
```

Comment:
ฝากเติม 
```suggestion
   func removeGradientBackground() {
        self.layer.sublayers?.forEach {
            if $0.isKind(of: CAGradientLayer.self) {
                $0.removeFromSuperlayer()
            }
        }
    }
```

เพิ่มไปด้วยครับ แล้วใช้ตอน prepareForReuse ใน UITableViewCell ด้วยครับ เพราะว่า cell มันจะเกิดการ resuse เราต้องเคลียร์มันออกไปด้วยครับ ไม่งั้นจะ leak